cost_analysis.py

```
@@ -0,0 +1,240 @@
+    print("=" * 50)
+    
+    # Get video data
+    db_path = "data/youtube_data.db"
+    videos = get_videos_with_durations(db_path)
+    
+    # Calculate total duration
+    total_seconds = sum(v['duration_seconds'] for v in videos if v['duration_seconds'])
+    total_minutes = total_seconds / 60
+    total_hours = total_minutes / 60
+    
+    print(f"\n📊 ESTADÍSTICAS DE DURACIÓN:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)